README.md

```diff
@@ -166,6 +166,8 @@ var nodes = jp.apply(data, '$..author', function(value) { return value.toUpperCa
 // ]
 ```
 
+If the supplied application function returns the special value `:delete:` the node will be deleted and not updated as is otherwise the default case.
```

Comment:
@kristianmandrup Why do you need special `:delete:` value you can simply detect when callback returns `undefined`.
\+ What if I want to replace the value with literal `:delete:`.
IMHO, it's bad practice to create magic values.